crates/typst-syntax/src/ast.rs

```diff
@@ -2006,8 +2015,67 @@ node! {
 
 impl<'a> ImportItems<'a> {
     /// The items to import from the module.
-    pub fn idents(self) -> impl DoubleEndedIterator<Item = Ident<'a>> {
-        self.0.children().filter_map(SyntaxNode::cast)
+    pub fn items(self) -> impl DoubleEndedIterator<Item = ImportItem<'a>> {
+        self.0.children().filter_map(|child| {
+            child
+                .cast::<RenamedImportItem>()
+                .map(ImportItem::Renamed)
+                .or_else(|| child.cast::<Ident>().map(ImportItem::Simple))
+        })
+    }
+}
+
+/// An imported item, potentially renamed to another identifier.
+#[derive(Debug, Copy, Clone, Hash)]
+pub enum ImportItem<'a> {
+    /// A non-renamed import (the item's name in the scope is the same as its
+    /// name).
+    Simple(Ident<'a>),
+    /// A renamed import (the item was bound to a different name in the scope
+    /// than the one it was defined as).
+    Renamed(RenamedImportItem<'a>),
+}
+
+impl<'a> ImportItem<'a> {
+    /// The original name of the imported item,
```

Comment:
could you wrap comments at 80 cols?

Reply:
sorry, missed it here lol.

(Fun fact: I believe `rustfmt` has a built-in parameter that handles this, but AFAIK you need to use nightly to be able to pass parameters to rustfmt. ;-;)